packages/grafana-prometheus/src/index.ts

```diff
@@ -85,3 +85,11 @@ export {
   type StandardPromVariableQuery,
 } from './types';
 export { PrometheusVariableSupport } from './variables';
+
+// For explore metrics
+export { default } from './language_provider';
+export { getPrometheusTime } from './language_utils';
+export { isValidLegacyName, utf8Support, wrapUtf8Filters } from './utf8_support';
```

Comment:
This is out of scope for this PR, but just want to mention that we are still importing from `'@grafana/prometheus/src/utf8_support'` here:

https://github.com/grafana/grafana/blob/f6194931f59382e4e9089d1e0a79936c9ccc1b3c/public/app/features/explore/PrometheusListView/RawListItem.tsx#L5

Reply:
Yeap that's outside of this PR. When I'll remove the barrel file from prometheus I'll deal this part too. But thanks for pointing this. I'll do a through cleaning :)